hazelcast/src/test/java/com/hazelcast/cache/CacheReadWriteThroughTest.java

```diff
@@ -60,47 +64,48 @@
     private HazelcastServerCachingProvider cachingProvider1;
     private HazelcastServerCachingProvider cachingProvider2;
 
+    protected Config createConfig() {
+        return new Config();
+    }
+
     @Before
-    public void init() {
+    public void setup() {
         factory = new TestHazelcastInstanceFactory(2);
```

Comment:
yes, we could. In fact, i had not looked at that part. Just renamed the method and added `createConfig()` to use same test at EE